scripts/dist.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-./dist-no-libs.sh
-mkdir -p $TO_DIR/libs/
-cp -r libs/ $TO_DIR/libs/
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
Feels like these scripts change in every PR 😅

Reply:
lol :D we're simply agile 😎